FirebaseAuth/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@
   `Auth.currentUser` API. This resolves some Firebase 11 issues where the
   current user is unexpectedly `nil` at startup.
 - [fixed] Restore Firebase 10 decoding behavior to prevent user provider data
```

Comment:
Would you clarify that these two fixes only impact apps migrating from Firebase 10 to 11

Reply:
Done–– ready for another pass.